topics/icrc_34_get_delegation.md

```diff
@@ -1,50 +1,54 @@
 
 <!-- TOC -->
-* [ICRC-34: Get Global Delegation](#icrc-34-get-global-delegation)
+* [ICRC-34: Delegation](#icrc-34-delegation)
   * [Summary](#summary)
   * [Method](#method)
   * [Scope (according to the ICRC-25 standard)](#scope-according-to-the-icrc-25-standard)
-    * [Example RPC Request Permission](#example-rpc-request-permission)
-  * [Request](#request)
-    * [Example RPC Request](#example-rpc-request)
-    * [Response](#response)
-    * [Example RPC Response](#example-rpc-response)
+    * [Example RPC Permission Request](#example-rpc-permission-request)
+  * [icrc-25-supported-standards](#icrc-25-supported-standards)
+  * [Request params](#request-params)
+    * [Example RPC Delegation Request](#example-rpc-delegation-request)
+  * [Result](#result)
+    * [Example RPC Delegation Response](#example-rpc-delegation-response)
   * [Message Processing](#message-processing)
   * [Errors](#errors)
 <!-- TOC -->
 
 ## Summary
 
-When a relying party wants to authenticate as a user, it uses
-a [session key](https://internetcomputer.org/docs/current/references/ic-interface-spec/#signatures), and
-below `icrc34_get_global_delegation` method to obtain a delegation chain that allows the session key to sign for the
-user's global identity. The obtained delegation chain **MUST** be restricted by the signer to canister targets that
-trust the relying party following the [ICRC-28](./icrc_28_trusted_origins.md) standard.
+When a relying party wants to remove wallet approval prompts and sign on behalf of its users, it uses the icrc34_delegation 
+method described below. This method generates a delegation chain, an object with a stable identifier, allowing the relying 
+party to sign transactions on behalf of the user.
 
-- Due to the canister target restriction, only the canisters that the relying party owns can be called on behalf of the user.
-- A new delegation chain must be obtained for every new canister created by the relying party.
-- The identity is the same across all relying parties, the user is not anonymous between different relying parties.
+The delegation chain's stable identifier can take two forms: an account that may include ICP, ICRC-1, and other assets 
+(Account Delegation), or an account designed exclusively for the relying party (Relying Party Delegation).
 
-## Method
+If a relying party wants to receive an Account Delegation, the `icrc34_delegation` request **MUST** include canisters it
+controls as `targets` where each canister **MUST** implement the `icrc28_trusted_origins` endpoint as per the 
+[ICRC-28](./icrc_28_trusted_origins.md) standard.
 
-**Name and Scope:** `icrc34_get_global_delegation`
+Relying Party Delegations **MUST** be restricted by the signer to individual relying parties.
 
-**Prerequisite:** Active session with granted permission scope `icrc34_get_global_delegation` or `*`.
+Signers **SHOULD** give users the choice to authenticate with their wallets (if available) or with an app-specific identifier.
```

Comment:
I would personally consider keeping this out of the spec, a signer could decide to not give the user choice but decide based on the type of delegation requested. Which in most signers seems a more straightforward implementation.

Basically giving a user the choice to return a RP delegation when an Account delegation is requested is an advanced implementation to allow for anonymity but is also has technical challenges like what to return in the accounts request etc.

I can understand the reasons for adding it to the spec, but I would at least change it to `COULD` since the main goal here is probably to indicate that signers _could_ return a RP delegation when a Account delegation was requested.

```suggestion
Signers **COULD** give users the choice to authenticate with their relying party specific identifier instead of accounts (if available).
```